airgun/entities/ansible_role.py

```diff
@@ -0,0 +1,57 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep
+from airgun.navigation import navigator
+from airgun.views.ansible_role import AnsibleRolesImportView
+from airgun.views.ansible_role import AnsibleRolesView
+
+
+class AnsibleRolesEntity(BaseEntity):
+    endpoint_path = '/ansible/ansible_roles'
+
+    def search(self, value):
+        """Search for existing Ansible Role"""
+        view = self.navigate_to(self, 'All')
+        return view.search(value)
+
+    def delete(self, entity_name, values):
+        """Delete Ansible Role from Satellite"""
+        view = self.navigate_to(self, 'All')
+        view.search(entity_name)
+        view.table.row(name=entity_name)['Actions'].widget.fill('Delete')
+        self.browser.handle_alert()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
+
+    def count_imported_roles(self):
```

Comment:
Not sure about naming for this in airgun, but I'd think about two things with this function:

1. It's a property (should be decorated, takes no arguments and doesn't change state)
2. I'd call it `imported_roles_count`, but this is kinda subjective and might go against the grain with other entity function names in airgun.